### PR TITLE
Nk/driver tele

### PR DESCRIPTION
**Pull request by nkrackow**
Small PR that implements the Driver telemetry.

### CODE REVIEW

File: src/bin/driver.rs [hunk 1]
```diff
@@ -119,6 +119,28 @@ impl Default for Settings {
     }
 }
 
+#[derive(Serialize, Copy, Clone, Default, Debug)]
+pub struct Monitor {
+    output_current: [f32; 2],
```
Comment (jordens) [inline src/bin/driver.rs:124]: Please check the names. Do you need `output` in all these (and elsewhere)? If there is `output`, why is there no `input` but `feedback`? `feedback` is also probably a bad name as that input it does not necessarily need to be for feedback at all.

File: src/bin/driver.rs [hunk 2]
```diff
@@ -119,6 +119,28 @@ impl Default for Settings {
...
+    output_voltage: [f32; 2],
+    cpu_temp: f32,
+    overtemp: bool,
+}
+
+#[derive(Serialize, Copy, Clone, Default, Debug)]
+pub struct LowNoise {
```
Comment (jordens) [inline src/bin/driver.rs:131]: `LowNoiseControl{adc0, current}`?

Try sticking to the names on the schematic/panel.
If this is a current sink, is `output_current` really a useful name? What about just `current`? It's clear from the context that this is not the monitor data.

**Review by jordens — CHANGES_REQUESTED** [review]
Just name bikeshedding.

File: src/bin/driver.rs (hunk 1 above)
Comment (nkrackow) [reply src/bin/driver.rs:124]: Ok, I mean yes there ate inputs on stabilizer. Those might not be exposed in telemetry but I thought just `voltage` might be confusing for someone who has not used Drive+Stabilizer before.

File: src/bin/driver.rs (hunk 2 above)
Comment (jordens) [reply src/bin/driver.rs:131]: What about the struct name?

File: src/bin/driver.rs (hunk 1 above)
Comment (jordens) [reply src/bin/driver.rs]: Then call the struct `DriverMonitor` to make that clear. And use two of them, not one with two arrays.

**Pull request merged**